library/core/src/alloc/layout.rs

```diff
@@ -4,6 +4,12 @@ use crate::mem;
 use crate::num::NonZeroUsize;
 use crate::ptr::NonNull;
 
+/// While this function is used in one place and its implementation
+/// could be inlined, the previous attempts to do so introduced
+/// runtime regression in rustc:
```

Comment:
```suggestion
/// compile time regression in rustc:
```

Reply:
Rustc got slower, so it is a runtime regression in rustc. Of course rustc is compiler so one could as well say that it is a compile time regression, but that could be interpreted to mean that more time is required to compile rustc itself, which is not the case, since bootstrap timings show a small improvement which the change applied.

Reply:
"run time regression" sounds too much like *running* `Layout::new::<T>()` becomes slower. Maybe just change the entire sentence to 

> the previous attempts to do so make rustc slower

?